### ISSUE TITLE
Missing GroupID in Unit Default QA test

### ISSUE THREAD

**Issue by ntknguyen** [US-EPA-CAMD/PI-1#5971]

**Comment by ntknguyen** [comment]
1. GroupID value is not showing in the global/workspace view.  User reported the value is not saved after he added it. 
2. Typo of the word 'emissions' in the [alert banner](https://ecmps-tst.app.cloud.gov/workspace/evaluate), 
[RT#73351](https://rt.camdsupport.com/rt/Ticket/Display.html?id=73351)
<img width="1153" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOU03QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a1ad1321a119c63ea4a4b6f4b95a79259568e999/MissingGroupIDinUnitDefault.png" alt="MissingGroupIDinUnitDefault.png" />
<img width="1739" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVU3QWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b2574d36e8bdb4d2e01cd7524c3703a65d353c08/FACT-UnitDefaultRecord.png" alt="FACT-UnitDefaultRecord.png" />

**Comment by maxdiebold-erg** [comment]
@ntknguyen Is it possible to migrate those images and ticket to GitHub?

**Comment by ntknguyen** [comment]
> [@ntknguyen](https://github.com/ntknguyen) Is it possible to migrate those images and ticket to GitHub?



![Image](https://github.com/US-EPA-CAMD/easey-ui/assets/17596531/2a4de571-432c-4408-8970-3b7be55dc1bf)
![Image](https://github.com/US-EPA-CAMD/easey-ui/assets/17596531/04fd8630-81b4-43d0-8564-0682b328560d)

**Comment by maxdiebold-erg** [comment]
@ntknguyen I believe the typo was fixed in [`easey-content:c194686`](https://github.com/US-EPA-CAMD/easey-content/commit/c194686f62cdc54110ce96cc9f36f2dd87ca4a83). Could you confirm this, or otherwise point out the existing typo?

**Comment by ntknguyen** [comment]
> [@ntknguyen](https://github.com/ntknguyen) I believe the typo was fixed in [`easey-content:c194686`](https://github.com/US-EPA-CAMD/easey-content/commit/c194686f62cdc54110ce96cc9f36f2dd87ca4a83). Could you confirm this, or otherwise point out the existing typo?

@maxdiebold-erg - It looks like it's fixed in dev not in test.  I guess it will reflect the changes in test later on, correct?

**Comment by maxdiebold-erg** [comment]
You are right, that change was not promoted to test or beta. ~I will do that~. It will be promoted along with this ticket.

**Comment by lgiannini1** [comment]
Confirmed on dev. Emissions is spelled correctly in Submit screen banner. Comment with Group ID testing results on ticket #5984.

**Comment by maheese** [comment]
Tested as part of #5984
